doc/api/globals.md

```diff
@@ -322,6 +322,19 @@ added: v0.0.1
 
 [`clearTimeout`][] is described in the [timers][] section.
 
+## `CloseEvent`
+
+<!-- YAML
+added: REPLACEME
+-->
```

Comment:
Should be marked as experimental?

```suggestion
## `CloseEvent`

<!-- YAML
added: REPLACEME
-->

> Stability: 1 - Experimental.
```

Reply:
I'd prefer to just rip the band-aid off and have this stable in v24 then having to make two different PRs for it to be stable.

Reply:
I don't see a reason to make it experimental. It uses undici's webidl implementation under the hood so it's highly unlikely to have any bugs. We test it against the WPTs and our own tests. The implementation of it is pretty simple, there isn't much room to get things wrong.

Reply:
That all makes sense to me, I was asking because from what I can tell the `WebSocket` global is declared as experimental 🙂

Reply:
It's coupled with WebSocket, `--no-experimental-websocket` will disable it!